Database/Models/Economy/TaxCredit.cs

```diff
@@ -0,0 +1,22 @@
+using System.Text.Json.Serialization;
+using System.ComponentModel.DataAnnotations.Schema;
+using SpookVooper_2.Database.Models.Entities;
+
```

Comment:
I can't even imagine how we would go about tac credits

Reply:
one idea could be pay them out, like add to the group/user balance, then taxes would take. Any leftover would be kept.